deploy/chart/crds/0000_50_olm_00-clusterserviceversions.crd.yaml

```diff
@@ -662,13 +662,19 @@ spec:
                       deployments:
```

Comment:
You should update the `/manifests` folder as well to make sure that the openshift install actually consumes these CRD changes.